po/ja.po

```diff
@@ -8527,10 +8527,13 @@ msgid ""
 "Rust support generics, which lets you abstract algorithms or data structures "
 "(such as sorting or a binary tree) over the types used or stored."
 msgstr ""
+"Rustはジェネリクス（generics）をサポートします。これにより、使用または保存す"
+"る型に関してアルゴリズムやデータ構造（ソートアルゴリズムや、二分木など）を抽"
+"象化することができます。"
 
 #: src/generics/data-types.md:3
 msgid "You can use generics to abstract over the concrete field type:"
-msgstr ""
+msgstr "ジェネリクスを使って、平面を表す具象型を抽象化することができます："
```

Comment:
This "field" is not "平面" but "a member of the struct"

Reply:
Thanks! The word "field" seems like a member variable or data memeber in this context. I'll transliterate the word as "フィールド" since it's a technical term :)